src/pages/issues.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useState, useEffect } from 'react';
+import useFetch from '@/hooks/useFetch';
+import IssueList from '../components/issues/IssueList';
+import classNames from '@/styles/issues.module.scss';
+import Layout from '@/components/Layout';
+import Head from '@/components/head';
+import {
+    ISSUES_FETCH_ERROR_MESSAGE,
+    NO_ISSUES_FOUND_MESSAGE,
+} from '@/components/constants/messages';
+import { ISSUES_URL } from '@/components/constants/url';
+
+const Issues: FC = () => {
+    const [issueList, setIssueList] = useState<[]>([]);
+
+    const { response, error, isLoading } = useFetch(ISSUES_URL, {
+        data: {
+            userData: {
+                roles: {
+                    super_user: true,
+                },
+            },
+        },
+    });
+
+    useEffect(() => {
+        if ('issues' in response) {
+            const issues = response.issues;
+            const onlyIssues = issues.filter(

```

Comment:
The naming here is not self-explanatory.

Reply:
Done.